apps/admin-gui/src/app/users/pages/user-detail-page/user-overview/user-overview.component.ts

```diff
@@ -139,46 +139,55 @@ export class UserOverviewComponent implements OnInit {
         }
       );
     }
-    this.navItems.push({
+    this.items.push({
       cssIcon: 'perun-attributes',
       url: `${urlStart}/attributes`,
       label: 'MENU_ITEMS.USER.ATTRIBUTES',
       style: 'user-btn',
     });
-    this.navItems.push({
+    this.items.push({
       cssIcon: 'perun-roles',
       url: `${urlStart}/roles`,
       label: 'MENU_ITEMS.USER.ROLES',
       style: 'user-btn',
     });
     if (this.isServiceUser) {
-      this.navItems.push({
+      this.items.push({
         cssIcon: 'perun-manager',
         url: `${urlStart}/associated-users`,
         label: 'MENU_ITEMS.USER.ASSOCIATED_USERS',
         style: 'user-btn',
       });
     } else {
-      this.navItems.push({
+      this.items.push({
         cssIcon: 'perun-service-identity',
         url: `${urlStart}/service-identities`,
         label: 'MENU_ITEMS.USER.SERVICE_IDENTITIES',
         style: 'user-btn',
       });
     }
-    this.navItems.push({
+    this.items.push({
       cssIcon: 'perun-ban',
       url: `bans`,
       label: 'MENU_ITEMS.USER.BANS',
       style: 'user-btn',
     });
-    if (!window.location.pathname.startsWith('/admin')) {
-      this.navItems.push({
-        cssIcon: 'perun-settings2',
-        url: `${urlStart}/settings`,
-        label: 'MENU_ITEMS.ADMIN.SETTINGS',
-        style: 'user-btn',
-      });
+
+    if (this.inMyProfile) {
+      this.items.push(
+        {
+          cssIcon: 'perun-settings2',
+          url: '/myProfile/settings/passwordReset',
+          label: 'MENU_ITEMS.USER.PASSWORD_RESET',
+          style: 'user-btn',
+        },
+        {
+          cssIcon: 'perun-settings1',
+          url: '/myProfile/settings/guiConfig',
```

Comment:
Maybe we can throw away `/settings` from this URL when these items are now on the same level?

Reply:
This change was reverted. Originally I thought it would not be possible to have not entity related items expandable, but it was fixed by merging the root and "normal" side menu item component.